### PR TITLE
Fix/gett 94497 unique jobs

### PR DESCRIPTION
**Pull request by Darkemon**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	@echo "+ $@"
+	docker-compose rm -fsv
+
+.PHONY: test-run
+test-run:
+	@echo "+ $@"
+	go test -v -p 1 -race -coverprofile=coverage.txt -covermode=atomic ./...
+
+.PHONY: test
+test: tearup test-run teardown
```
Comment (sergeyslonimsky) [inline Makefile]: @Darkemon do we need to add linter here as well, wdyt?

Comment (Darkemon) [reply Makefile]: I thought about that too, but it'll take take to make the code to conform the linter rules. I suggest to leave it as is to not spend time.

**Review by sergeyslonimsky — APPROVED** [review]

File: worker.go [hunk 2]
```diff
@@ -280,22 +304,30 @@ func (w *worker) removeJobFromInProgress(job *Job, fate terminateOp) {
 
 type terminateOp func(conn redis.Conn)
 
+type opType int
+
+const (
+	opTerminate = opType(iota)
```
Comment (shilkin) [inline worker.go:312]: Less verbose would be
```golang
const (
	opTerminate = opType(iota)
	opRetry
	opDead
)
```

File: worker.go [hunk 3]
```diff
@@ -280,22 +304,30 @@ func (w *worker) removeJobFromInProgress(job *Job, fate terminateOp) {
 
 type terminateOp func(conn redis.Conn)
 
+type opType int
```
Comment (shilkin) [inline worker.go:309]: Let's write a comment to this.

Comment (Darkemon) [reply worker.go:309]: Added [bda8c86](https://github.com/GettEngineering/work/pull/3/commits/bda8c862c12ddaa4f1de9ec0bee0070008168fd6)

File: worker.go (hunk 2 above)
Comment (Darkemon) [reply worker.go:312]: Fixed [bda8c86](https://github.com/GettEngineering/work/pull/3/commits/bda8c862c12ddaa4f1de9ec0bee0070008168fd6)

**Review by shilkin — APPROVED** [review]

**Pull request merged**